<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="teStrToSend">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>311</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSendToClient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Send to Web Client</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teStrRecFromClient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>311</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Received From Web Client</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
